controller/src/beerocks/bml/bml.cpp

```diff
@@ -754,3 +754,16 @@ int bml_client_get_client(BML_CTX ctx, const char *sta_mac, struct BML_CLIENT *c
     auto pBML = static_cast<bml_internal *>(ctx);
     return pBML->client_get_client(tlvf::mac_from_string(std::string(sta_mac)), client);
 }
+
+//================================================
+int bml_oren_reverse_string(BML_CTX ctx, const char *str_in, char *str_out)
+{
+    LOG(TRACE) << "=========> cli_bml::bml_oren_reverse_string(): " << str_in << std::endl;
+
+    if (!ctx || !str_in || !str_out)
+        return (-BML_RET_INVALID_ARGS);
```

Comment:
[coding style ](https://github.com/prplfoundation/prplMesh/blob/master/CODINGSTYLE.md) 

```suggestion
    if (!ctx || !str_in || !str_out) {
        return (-BML_RET_INVALID_ARGS);
    }
```

Reply:
fixed